tests/behaviour/steps/cubes.py

```diff
@@ -119,7 +119,7 @@ def step_impl(context):
             data_file,
             config_file,
             output_directory=context.out_dir,
-            validation_errors_file_out=context.out_dir / "validation_errors.json",
+            validation_errors_file_name=context.out_dir / "validation_errors.json",
```

Comment:
Shouldn't this just be the name of the file rather than including the out directory too?